mitmproxy/addons/replace.py

```diff
@@ -72,6 +72,10 @@ def configure(self, options, updated):
                     raise exceptions.OptionsError(
                         "Invalid regular expression: %s - %s" % (rex, str(e))
                     )
+                if s.startswith("@") and not os.path.isfile(s[1:]):
```

Comment:
I think we should use os.path.expanduser here since we are doing that in the editor and also in the replace function.